common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -464,6 +525,21 @@ innovation_troll_tactics = {
 	flag = global_maa
 }
 
+innovation_tuskarr_tactics = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @trapper
+
+	potential = {
+		this.culture_group = culture_group:tuskarr_group
+	}
+
+	unlock_maa = trapper
+	unlock_maa = turtle_rider
```

Comment:
I would say they are too expensive.

Reply:
Who turtle riders?

Reply:
> Who turtle riders?

Yes.

> We can make them light cavalry if you think that's smart

No. I see you make cultural MaAs 20% more expensive than the basic MaA. But I wouldn't stick to it so much. Cost may be 120-80% of the basic MaA. Especially when culture can't allow so much money.

Reply:
> Making them weaker would throw the balance of, since all other maas are 119%-121% of base maa.

I said nothing about making them weaker.

Reply:
You make them 80% of the base MaA cost, 120% of the balance factor.